.gitlab-ci.yml

```diff
@@ -385,6 +388,7 @@ doc:refman:dune:
   artifacts:
     paths:
       - _build/default/doc/sphinx_build/html
+    expire_in: 2 months
```

Comment:
Why not the same as dune-ci-template?

Reply:
I used 2 months for docs and 1 month for binaries. I dunno, these values are arbitrary.

Reply:
>I used 2 months for docs and 1 month for binaries. I dunno, these values are arbitrary.

Please pick one and use it for both.

Reply:
No need to repeat it when it's the same as the template.

Reply:
well seen, let's see now.